python/ray/actor.py

```diff
@@ -472,11 +472,23 @@ def _remote(self,
                              "Please use Actor._remote(name='some_name') "
                              "to associate the name.")
 
+        if name and not detached:
```

Comment:
These checks seem to allow `detached=True` and `name=None`. Is that intended?

Reply:
There's a check for that above.